### PR TITLE
maintain: Chapter 1

### LINKED ISSUE #532: maintain: Chapter 1
## Getting Started
- [x] Add a brief introduction to the content covered in this chapter

## Installation
- [x] Rewrite parts of the installation section (add a few lines), to put forward the flow of installation with asdf instead of the direct Scarb installation. This is extremely practical when working on multiple projects with different versions. Keep a link to the Scarb docs to install normally.
- [x] update Scarb –version output to 2.5.0 for now

## Hello World
- [x] remove “and a directory for the ‘’Hello, World!” project within the cairo_projects directory” as it will be created with scarb just after
- [x] reorder the Scarb Note after the command line instructions
- [x] Navigate to your **cairo_projects** directory
- [x] “--vcs” flag should be “--no-vcs”
- [x] the name and the version of Scarb to use => the name of the package and the version of Scarb to use
- [x] modify the Scarb.toml file as edition is now by default “edition = 2023_11” 
- [x] Remove references to `scarb test` - out of scope for now
- [x] Add ‘’scarb cairo-run’’ in the recap of commands learned
- [x]  Remove scripts and `scarb run` references

### PR DESCRIPTION
**Pull request by TAdev0**
Closes #532

- - -
This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/535)

### CODE REVIEW

**Comment by TAdev0** [comment]
@enitrat  there is still this issue when we run the main function through the book,  macros are not recognized: 

```
fn main() {
    println!("Hello, World!");
}
 error: Inline macro `println` not found.
 --> astro.cairo:2:5
    println!("Hello, World!");
    ^***********************^
```

**Comment by TAdev0** [comment]
Also removed the unnecessary `--available-gas` here.

**Comment by enitrat** [comment]
It's because the wasm version is outdated nothing we can do yet

File: src/ch01-01-installation.md [hunk 1]
```diff
@@ -20,21 +20,35 @@ Scarb requires a Git executable to be available in the `PATH` environment variab
 
 To install Scarb, please refer to the [installation instructions](https://docs.swmansion.com/scarb/download). We strongly recommend that you install
 Scarb [via asdf](https://docs.swmansion.com/scarb/download.html#install-via-asdf), a CLI tool that can manage multiple language runtime versions on a per-project basis.
-This will ensure that the version of Scarb you use to work on a project always matches the one defined in the project settings, avoiding problems lead to version mismatch.
+This will ensure that the version of Scarb you use to work on a project always matches the one defined in the project settings, avoiding problems that lead to version mismatches.
```
[the remark this answers is not in the document]
Comment (TAdev0) [reply src/ch01-01-installation.md:23]: better indeed

**Pull request merged**